internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,36 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
-)
 
-var (
-	StepLength = 0.65 // длина шага в метрах
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
+var StepLength = 0.65 // длина шага в метрах
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	pieces := strings.Split(data, ",")
+	if len(pieces) != 2 {
+		return 0, 0, errors.New("Incorrect data received")
+	}
+
+	steps, err1 := strconv.Atoi(pieces[0])
```

Comment:
Ошибку можно использовать одну, они не пересекаются
```suggestion
	steps, err := strconv.Atoi(pieces[0])
```